example_config.yaml

```diff
@@ -4,6 +4,12 @@
 # Authentication #
 ##################
 
+# User Access Token
+access_token_id: TODO
+access_token_secret: TODO
+
+# OR
+
 # Client Key Pair
```

Comment:
It would be helpful to comment out the client key pair in the example config, so that a user could just fill out the access token elements and use it. 

It might also be helpful to add an `auth_method` key to this file, with a comment about what it should be set to, if the file contains multiple credentials.